static/css/perdiem.css

```diff
@@ -176,9 +176,21 @@ h2.profile-header {
     font-size: 25px;
 }
 
-p.profile-total {
+h5.profile-total {
     text-align: center;
-    font-size: 25px;
+    font-size: 18px;
+    padding-top: 5px;
+    padding-left: 20px;
+    padding-right: 20px;
+    font-family: Dosis;
```

Comment:
It's generally good practice to offer fallback fonts in the CSS for the time while the font is loading. ie:
```
font-family: Dosis, sans-serif;
```